omod-common/src/main/java/org/openmrs/module/webservices/rest/OpenmrsPathMatcher.java

```diff
@@ -31,7 +31,7 @@ public boolean match(String pattern, String path) {
 	
 	@Override
 	public boolean matchStart(String pattern, String path) {
-		return this.matchStart(pattern, path);
+		return this.delegate.matchStart(pattern, path);
```

Comment:
hi,@dkayiwa  @mks-d what could be your take on the changes made here for the ticket

Reply:
cc @ibacher